deb_files/hornet.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Hornet
+
+[Service]
+WorkingDirectory=/var/run/hornet
+Restart=always
+ExecStartPre=/usr/bin/hornet_download_snapshot.sh
+ExecStart=/usr/bin/hornet -c /etc/hornet.conf
```

Comment:
isn't it `/etc/gohornet.conf` ?

Reply:
Should be /etc/gohornet/gohornet.conf 
Also Timeoutsec=Infinity should be set to let hornet gracefully shutdown.